Portal/src/Datahub.Infrastructure/ConfigureServices.cs

```diff
@@ -47,16 +46,20 @@ public static class ConfigureServices
 
         if (configuration.GetValue<bool>("ReverseProxy:Enabled"))
         {
-            services.AddDatahubReverseProxyServices();
+            services.AddTransient<IReverseProxyConfigService, ReverseProxyConfigService>();
+            services.AddSingleton<IProxyConfigProvider, ProxyConfigProvider>();
+            services.AddSingleton<IReverseProxyManagerService, ReverseProxyManagerService>();
         }
 
         services.AddHostedService<PreloaderService>();
         services.AddMemoryCache();
 
+        var whereAmI = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
+        var isDevelopment = !string.IsNullOrEmpty(whereAmI) && whereAmI.ToLower() == "development";
 
         // in Development, using InMemory MassTransit transport, HealthCheckConsumer and file system (FileWatcherService)
         // to pass and process HealthCheck messages
-        if (DevTools.IsDevelopment())
```

Comment:
this is rolling back some of the changes I did - I tried to centralize the tests for dev env instead of having them all over with string